server/deploy.go

```diff
@@ -99,6 +99,14 @@ func getDatabaseMatrixFromDeploymentSchedule(schedule *api.DeploymentSchedule, b
 		for _, id := range matchedDatabaseList {
 			stage = append(stage, databaseMap[id])
 		}
+		// sort databases in stage based on names.
```

Comment:
The name here is an unfortunate because we do have a stage concept. At first glance, I am confused why we need to sort the stage and then realize we are sorting the database name.

I would prefer to just name it as databaseList and rename pipeline as databaseMatrix.

Reply:
Done